code/modules/clothing/masks/hailer.dm

```diff
@@ -71,9 +72,9 @@ GLOBAL_LIST_INIT(hailer_phrases, list(
 	aggressiveness = AGGR_SHIT_COP
 	flags_inv = HIDEFACIALHAIR | HIDEFACE | HIDEEYES | HIDEEARS | HIDEHAIR | HIDESNOUT
 	visor_flags_inv = 0
-	tint = 1.5
 	flags_cover = MASKCOVERSMOUTH | MASKCOVERSEYES | PEPPERPROOF
```

Comment:
as long as it has pepperproof, and cover eyes, it must have fov

Reply:
look two lines south

Reply:
i am clinically insane